apps/next-evans/lib/get-custom-code.ts

```diff
@@ -0,0 +1,36 @@
+import { LANG } from './env';
+
+export default function getCustomCode(
+  friendSlug: string,
+  documents: string[],
```

Comment:
i shoulda mentioned this when we were pairing, or in the other PR, but i prefer to be explicit with parameter names that refer to string attributes of friends and documents, so here `documentSlugs` is better, because reading the signature from the outside, i'm not sure what `documents` means exactly